utils.js

```diff
@@ -1,5 +1,10 @@
 import dotiw from "dotiw";
 import { Location, Zmanim, HDate } from "@hebcal/core";
+import readline from "readline-sync";
+
+const rules = ["Gra", "MGA", "MGA 16.1"];
+const index = readline.keyInSelect(rules, "select your minhag");
+const rule = rules[index]
```

Comment:
I prefer to keep `utils.js` only for functions, and interactions with the user ("side effects") in index.js. 

To accomplish that, add a `rule` argument to the `zmanimOptions` and the `getZmanimForDay` functions. Move these 3 lines and the readline import into `index.js`, and pass the rule the user enters to the `getZmanimForDay` call, and then in `getZmanimForDay` pass it along to the `zmanimOptions` call.

Reply:
I did it like this, but it sounded more complex that i pass it 2 times, but if you think its better I'll change it back

Reply:
changed it